core/src/banking_stage/consumer.rs

```diff
@@ -519,10 +519,16 @@ impl Consumer {
         // once feature `apply_cost_tracker_during_replay` is activated, leader shall no longer
         // adjust block with executed cost (a behavior more inline with bankless leader), it
         // should use requested, or default `compute_unit_limit` as transaction's execution cost.
-        if !bank
+        if bank
```

Comment:
flipped the if here to be positive instead of negative, since we want to record the unadusted costs into our histogram no.